int-test-backend-server/scripts/spinup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
Maybe a comment here with an example of values to pass for things like `security-groups`?